functionnal_tests/cypress/integration/admin/navigation_admin_workspace.js

```diff
@@ -9,6 +9,7 @@ describe('navigation :: admin > workspace', function () {
     cy.visit('/ui')
   })
   it('', function () {
```

Comment:
no test description?
(change not required since it's unchanged code, but it'd be nice to avoid these things)

Reply:
Totally agree :smile: this test has 4 years, it should be reviewed at some point

Reply:
indeed